web/static/js/components/surveys/SurveyWizardRespondentsStep.jsx

```diff
@@ -48,6 +48,23 @@ class SurveyWizardRespondentsStep extends Component {
     this.props.actions.clearInvalids()
   }
 
+  invalidEntriesText(invalidEntries, invalidEntryType) {
+    const { t } = this.props
+    const filteredInvalidEntries = invalidEntries.filter(invalidEntry => invalidEntry.type == invalidEntryType)
+
+    if (filteredInvalidEntries.length == 0) return null
+
+    const lineNumbers = filteredInvalidEntries.slice(0, 3).map((entry) => entry.line_number)
+    const entryType = () => {
+      if (invalidEntryType == 'invalid-phone-number') return 'phone number'
+      else if (invalidEntryType == 'invalid-respondent-id') return 'respondent id'
+    }
+    const text = t('An invalid {{entryType}} was found at line {{lineNumbers}}', {lineNumbers: lineNumbers.join(', '), entryType: entryType()})
```

Comment:
I know this was as is before, but it would be better to have both entries as a whole

- `An invalid {{entryType}} was found at line {{lineNumbers}}`
- `An invalid {{entryType}} was found at line {{lineNumbers}} and {{count}} more.`

concatenating localized texts is not good. There is no context for `and {{count}} more.` for translating, and is not always the case that concatenating two individual translations leads to a valid phrase in the target language.